docs/en/ingest-management/processors/processors.asciidoc

```diff
@@ -0,0 +1,52 @@
+[[elastic-agent-processor-configuration]]
+= Define processors
+
+//REVIEWERS: I think we need to define "processors." Here's my first shot.
+//I think we also need to make it clear that these are different from the
+//processors defined in ingest pipelines. Should we qualify the name?
+
+Processors are lightweight processing components that you can use to parse,
+filter, transform, and enrich data at the source. For example, you can use
+processors to:
+
+* reduce the number of exported fields
+* enhance events with additional metadata
+* perform additional processing and decoding
+* sanitize data
+
+Each processor receives an event, applies a defined action to the event, and
+returns the event. If you define a list of processors, they are executed in the
+order they are defined.
+
+[source,yaml]
+----
+event -> processor 1 -> event1 -> processor 2 -> event2 ...
+----
+
+[discrete]
+[[where-valid]]
+== Where are processors valid?
+
+The processors described in this section are valid:
+
+* Under integration settings in the Integrations UI in {kib}.
+* Under input configuration settings for standalone {agent}s. You can define
+processors that apply to all inputs defined in the configuration, or specific
+inputs.
```

Comment:
Could we add examples of where to put processors for these, such as a screenshot of processors in the integration editor and a section of standalone config yaml?

Reply:
Is it also worth noting that not all integrations support processors? for example Endpoint doesn't support them all right now.

Reply:
@joshdover Yes, I can note that, but this statement begs the question: How can users tell which integrations support processors? Do we document this somewhere?

Reply:
I've added more info. Let me know if it's good enough as is.

Reply:
LGTM. AFAIK it's not documented anywhere. The only way to easily tell is by looking at the integration editor for each data stream. In general, we've made an effort to make sure they're available on all of the most popular integrations. cc @nimarezainia in case there's anything I'm missing here